backend/src/scraper/bestbuy_scraper.py

```diff
@@ -51,15 +51,22 @@ def _product_specs(product_id: str) -> Dict:
                     { "name": s.get("name"), "value": s.get("value") } for s in spec_highlights
                 ],
             }
+        
+    @staticmethod    
+    def _pull_reviews(i: int, sku: int, params: Dict) -> List[str]:
+        search_sku = requests.get(f"https://www.bestbuy.ca/api/v2/json/reviews/{sku}?page={i}&source=us", headers = params.get('headers'))
```

Comment:
I just realized (which is probably also why the Best Buy products returned from the Google Shopping API can't be found in the Best Buy API), but for the Best Buy API we're using `.ca` (Canada) while the Google Shopping API is using `.com` (US).

Not sure what the best approach is to fix this, but just thought I should point it out.

Reply:
We're only using the Canadian site for reviews. The product information and specifications are coming from the US site. That's why I'm calculating title similarity in this script and why there are often no peer reviews for products.

Reply:
The US Best Buy API doesn't provide any review services. I filtered this Canadian Best Buy API on source=us (vs. source=can) but I'm not sure if it does much.